### ISSUE TITLE
mobile screen menu problem

### ISSUE THREAD

**Issue by basantshawpitagnent** [reactjs/reactjs.org#5062]
Unable to see submenu under menu in mobile (small screen). Suppose we selected advanced guides under DOCS from the footer. Only we can see the first item which **Accessibility**. Unable to navigate over other menus under advance guide.

**Comment by gurnoor-garry** [comment]
Can anyone please elaborate this issue
